src/modules/design/property-panel/indextabs/general/sections/pool/pool.html

```diff
@@ -25,10 +27,21 @@
         <tr>
           <th>Name</th>
           <td>
-            <input type="text" class="props-input" value.bind="businessObjInPanel.processRef.name" change.delegate="updateName()" disabled.bind="!isEditable">
+            <input type="text" class="props-input" value.bind="businessObjInPanel.processRef.name" change.delegate="publishDiagramChange()" disabled.bind="!isEditable">
           </td>
         </tr>
       </table>
     </div>
   </div>
+
+  <modal show.bind="showModal"
+         header-text="Warning: Editing Process ID">
+    <template replace-part="modal-body">
+      Changing the process ID could break the ProcessEngine where the diagram is going to deploy!<br>
```

Comment:
```suggestion
      Changing a process ID can potentially break your deployments! Only perform this action if you know what you are doing!<br>
```

Warum ist hier ein `<br>` am Ende?